### PR TITLE
fix(api-1645/1646/1647/1648/1649): update postman collection

### PR DESCRIPTION
**Pull request by dFayet**

### CODE REVIEW

File: content/files/Akeneo PIM API.postman_collection.json [hunk 1]
```diff
@@ -215,6 +215,50 @@
                 "rest",
                 "v1",
                 "attributes"
```
Comment (benjaminleneve) [inline content/files/Akeneo PIM API.postman_collection.json:217]: There are missing query parameters on this endpoint GET /api/rest/v1/attributes/{code}
- with_table_select_options

File: content/files/Akeneo PIM API.postman_collection.json [hunk 2]
```diff
@@ -706,6 +782,56 @@
                 "rest",
                 "v1",
                 "categories"
```
Comment (benjaminleneve) [inline content/files/Akeneo PIM API.postman_collection.json:784]: There are missing query parameters on this endpoint GET /api/rest/v1/categories/{code}
- with_position

File: content/files/Akeneo PIM API.postman_collection.json (hunk 1 above)
Comment (dFayet) [reply content/files/Akeneo PIM API.postman_collection.json:217]: Aren't you referencing to https://github.com/akeneo/pim-api-docs/pull/599/files#diff-965e2072337a3d3d1d8e0ba37d6d642f21cbaf9d9c009c75a6fa016b25516cc3R257 ?

File: content/files/Akeneo PIM API.postman_collection.json (hunk 2 above)
Comment (dFayet) [reply content/files/Akeneo PIM API.postman_collection.json:784]: Same  question here, isn't it https://github.com/akeneo/pim-api-docs/pull/599/files#diff-965e2072337a3d3d1d8e0ba37d6d642f21cbaf9d9c009c75a6fa016b25516cc3R830 ?

File: content/files/Akeneo PIM API.postman_collection.json (hunk 1 above)
Comment (benjaminleneve) [reply content/files/Akeneo PIM API.postman_collection.json:217]: Yes it is this option but it is also available for the GET of an attribute

File: content/files/Akeneo PIM API.postman_collection.json (hunk 2 above)
Comment (benjaminleneve) [reply content/files/Akeneo PIM API.postman_collection.json:784]: Yes it is this option but it is also available for the GET of a category

**Review by tfehringer — APPROVED** [review]

**Review by LevFlavien — APPROVED** [review]

**Pull request merged**